configs/braze.json

```diff
@@ -0,0 +1,30 @@
+{
+  "index_name": "braze",
+  "start_urls": [
+    {
+      "url": "https://www.braze.com/(?P<section>.*?)/",
+      "variables": {
+        "section": [
+          "documentation",
+          "academy"
+        ]
+      }
+    },
+    "https://www.braze.com/documentation/Platform_Wide",
+    "https://www.braze.com/academy/Getting_Started"
+  ],
+  "scrap_start_urls": false,
+  "stop_urls": [
+    ".*?[^/]$"
+  ],
+  "selectors": {
+    "lvl0": "ul.nav a.heading",
+    "lvl1": ".article h1",
+    "lvl2": ".article h2",
+    "lvl3": ".article h3",
+    "lvl4": ".article h4",
+    "text": ".article p, .article li"
+  },
+  "nb_hits": 8625,
```

Comment:
This will depend of your indexing. Should not be considered as parameter but as a outcome of scraping.